app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/provision_workflow.rb

```diff
@@ -12,12 +12,6 @@ class ManageIQ::Providers::IbmCloud::VPC::CloudManager::ProvisionWorkflow < ::Mi
 
   # Class methods. Do not move to sub module.
   class << self
-    # Specify the name of the provision dialog to pull from the database.
-    # @return [String] The :name field in the provision dialog.
-    def default_dialog_file
-      'miq_provision_vpc_dialogs'
-    end
```

Comment:
So even though we had this here the dialog name returned by automate takes precedence https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_request_workflow.rb#L651

It isn't clear to me how this worked previously...